src/main/java/org/dependencytrack/resources/v1/vo/DependencyGraphResponse.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * This file is part of Dependency-Track.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Steve Springett. All Rights Reserved.
+ */
+
+package org.dependencytrack.resources.v1.vo;
+
+import java.io.Serializable;
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * Response-Object that represent light component or service for rendering DependencyGraph
+ *
+ * @author Nathan Mittelette <mittelette.nathan@gmail.com>
+ * @since 4.9.0
+ */
+public final class DependencyGraphResponse implements Serializable {
```

Comment:
Nit: You can replace this class with a [`record`](https://docs.oracle.com/en/java/javase/17/language/records.html). DataNucleus *should* be smart enough to use the `record` constructor when populating results.